strongbox-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox-authentication-providers.yaml

```diff
@@ -0,0 +1,37 @@
+strongbox:
```

Comment:
:+1: 

Comments from @steve-todorov : I love it! :D

Reply:
I knew he would like it :wink: